docs/book/usage/web-api.md

```diff
@@ -129,6 +129,126 @@ The aggregator recognizes the attribute keys `content-disposition`, `content-enc
 `content-language`, `content-location`, `content-type`, and `link`, and when present returns
 the values in the corresponding HTTP headers.
 
+### Quilt HTTP APIs
+
+Walrus supports storing and retrieving multiple blobs as a single unit called a [quilt](./quilt.md).
+The publishers and aggregators support *quilt* related operations.
+
+**Note** Quilt APIs are supported with Walrus *1.29* and higher.
+
+```admonish warning
+The identifiers must be unique within a quilt.
+```
+
+#### Storing quilts
+
+```admonish tip
+All the query parameters available for storing regular blobs (see [Store](#store) section above) can
+also be used when storing quilts.
+```
+
+Use the following publisher API to store multiple blobs as a quilt:
+
+```sh
+# Store two files `document.pdf` and `image.png`, with custom identifiers `contract-v2` and
+# `logo-2024, # respectively.
+curl -X PUT "$PUBLISHER/v1/quilts?epochs=5&quilt_version=V1" \
```

Comment:
Remark: The `quilt_version` parameter is used but not described. Maybe we can remove it to avoid confusion? We can add it later when we actually have multiple versions.